node/subscription.go

```diff
@@ -54,22 +51,96 @@ func (s *subscription) Unsubscribe(ctx context.Context) error {
 	}
 
 	if response.Error != nil {
-		return errors.Errorf("%v", response.Error)
+		return errors.Errorf("%s", string(*response.Error))
```

Comment:
minor: when testing I noticed that without this string conversion you end up with an error message like:

```
2020/02/27 15:00:55 [FATAL] error unsubscribing logs: &[123 34 99 111 100 101 34 58 45 51 50 48 48 48 44 34 109 101 115 115 97 103 101 34 58 34 115 117 98 115 99 114 105 112 116 105 111 110 32 110 111 116 32 102 111 117 110 100 34 125]
```

Rather than the much more legible:

```
2020/02/27 15:18:13 [FATAL] error unsubscribing logs: {"code":-32000,"message":"subscription not found"}
```

:)